scripts/scraper-ng/test/ingredient-data.formal_syntax.test.js

```diff
@@ -0,0 +1,73 @@
+const {
+  process,
+  expectPositionElement,
+  expectNullPosition,
+} = require("./framework/utils");
+
+describe("data.formal_syntax", () => {
+  const ingredientName = "data.formal_syntax";
+  const recipe = { body: [ingredientName] };
+
+  test("valid", () => {
+    const valid = `<h2 id="Formal_syntax">Formal syntax</h2>
+                   <pre class="syntaxbox notranslate">{{ csssyntax("display") }}</pre>`;
+    const file = process(valid, recipe);
+
+    expect(file).not.hasMessageWithId(ingredientName);
+    expectPositionElement(file.data.ingredients[0], ingredientName, "h2");
+  });
+
+  test("missing h2", () => {
+    const file = process("", recipe);
+
+    expect(file.messages.length).toBeGreaterThan(0);
```

Comment:
I don't really understand the purpose of these `toBeGreaterThan` checks. Here (and in any other error case where the argument is 0) it's redundant, since we immediately afterwards check for a particular message.